### ISSUE TITLE
Allow specifying target

### ISSUE THREAD

**Issue by jmccann** [jmccann/drone-terraform#27]
Allow specifying the `--target` for plan/apply.

```
  -target=resource    Resource to target. Operation will be limited to this
                      resource and its dependencies. This flag can be used
                      multiple times.
```

**Comment by edwinavalos** [comment]
#30 Is my attempt at this.

**Comment by edwinavalos** [comment]
@jmccann I believe this issue can be closed now that #30 was merged.
